config/config.yml

```diff
@@ -3,11 +3,13 @@
 # (it will be ignored by git). For more info, you can read the dedicated page
 # here: http://port.us.org/docs/Configuring-Portus.html.
 
-# Settings for the Portus mailer.
+# Settings for the Portus mailer.a
+# Thenreply_to field is empty, as the user has the possibility to enable
+# it optionally by filling in the appropiate reply address.
```

Comment:
Move this comment to the option itself. as it's done for the `smtp` option for example.